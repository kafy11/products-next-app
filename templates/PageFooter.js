const PageFooter = () => <footer className="small-text">Made by Kayo Silva</footer>;

export default PageFooter;