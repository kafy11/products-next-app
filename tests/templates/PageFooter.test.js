import { shallow } from 'enzyme';
import PageFooter from '../../templates/PageFooter';

describe('PageFooter', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<PageFooter />);
        expect(wrapper).toMatchSnapshot();
    });
});