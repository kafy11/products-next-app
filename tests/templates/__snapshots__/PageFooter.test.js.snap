// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter should render correctly 1`] = `
<footer
  className="small-text"
>
  Made by Kayo Silva
</footer>
`;
